api/app/signals/apps/api/serializers/signal.py

```diff
@@ -513,7 +513,7 @@ def create(self, validated_data):
         )
 
         if attachments:
-            Signal.actions.copy_attachments(data=attachments, signal=signal)
+            Signal.actions.copy_attachments(data=attachments, signal=signal, created_by=logged_in_user.email)
```

Comment:
Here the user who creates the deelmelding is considered the one who adds the attachments. When done in a celery task this is none. Could this leads to any problems?